src/components/container/EhrNewsFeed/EhrNewsFeed.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useState } from 'react';
+import OnVisibleTrigger from '../../presentational/OnVisibleTrigger';
+import { getNewsFeedPage } from '../../../helpers/news-feed/data';
+import { format, parseISO } from 'date-fns';
+import { Action, Card, LoadingIndicator, Title } from '../../presentational';
+import language from '../../../helpers/language';
+import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import "./EhrNewsFeed.css"
+import { EhrNewsFeedEventModel, EhrNewsFeedFeed } from '../../../helpers/news-feed/types';
+import { previewFeed } from '../../../helpers/news-feed/previewData';
+import { eventTypeDefinitions } from '../../../helpers/news-feed/eventTypeDefinitions';
+
+export interface EhrNewsFeedProps {
+    previewState?: "default" | "procedures" | "labReports" | "immunizations" | "reports"
+    onEventSelected(eventReference: EhrNewsFeedEventReference): void
+    feed: EhrNewsFeedFeed
+    onReportSelected(reportID: string): void
+}
+
+export interface EhrNewsFeedEventReference {
+    feed: EhrNewsFeedFeed
```

Comment:
FeedFeed is awkward and looks like a mistake. Maybe EhrNewsFeedType? Also might consider not pluralizing these. Could we maybe just use the EhrNewsFeedEventTypes here?

Reply:
Type definitions are really supposed to be singular (assuming they represent a singular concept)...like any class or object.  Not sure why we would want to pluralize it?

But yeah I like FeedType better, updated to that

Reply:
I was talking about the EhrNewsFeedEventType enum values: 
"procedure" | "labReport" | "immunization" | "report"
Not renaming the type.

Reply:
Oh - those are actual parameters sent to the server.  Renaming them would actually require a server config change etc. (duplicating a lot of config probably, because it would need to be backwards compatible)...so not something worth doing I think

Reply:
Makes sense. Wasn't sure of those constraints.